doc/src/site/sphinx/configuration/configuration_properties.rst

```diff
@@ -363,6 +363,10 @@ External backend configuration properties
 |                                                       |                | the indefinite timeout. The unit is |
 |                                                       |                | milliseconds.                       |
 +-------------------------------------------------------+----------------+-------------------------------------+
+| ``spark.ext.h2o.external.hadoop.executable``          | ``hadoop``     | Name or path to hadoop binary       |
```

Comment:
`path to hadoop` -> `path to a hadoop executable`?

Reply:
Thanks!